public_dropin_apps_environments/python311_node22/env_info.json

```
@@ -0,0 +1,9 @@
+{
+  "id": "6807b77d70d573235153b370",
+  "name": "[Experimental] Python 3.12 + NodeJS 22.15 Base ENV",
```

This should be 3.11 right?

```suggestion
  "name": "[Experimental] Python 3.11 + NodeJS 22.15 Base ENV",
```